### PR TITLE
add 1.11 to compatibility test and update comment

### PR DESCRIPTION
**Pull request by wilsonwang371**
## Why are these changes needed?

add ray 1.11 to compatibility test

## Related issue number

#185 

## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [X] This PR is not tested :(

### CODE REVIEW

File: .github/workflows/test-job.yaml [hunk 1]
```diff
@@ -85,6 +73,20 @@ jobs:
         working-directory: ./cli
         args: --timeout=3m
         skip-go-installation: true
+        skip-pkg-cache: true
+
+    - name: Install goimports
```
Comment (Jeffwan) [inline .github/workflows/test-job.yaml:78]: any reason to move imports after linter?  @wilsonwang371

**Comment by wilsonwang371** [comment]
This also fixed golang lint error message.

@Jeffwan please take a look

File: .github/workflows/test-job.yaml (hunk 1 above)
Comment (wilsonwang371) [reply .github/workflows/test-job.yaml:78]: This fixed golang lint error message we saw in previous runs

Comment (Jeffwan) [reply .github/workflows/test-job.yaml:78]: Got it

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**